xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-notifiers/xwiki-platform-notifications-notifiers-api/src/main/java/org/xwiki/notifications/notifiers/script/NotificationNotifiersScriptService.java

```diff
@@ -225,7 +225,9 @@ private String getAsyncNotification(NotificationAsyncRendererConfiguration confi
             if (result != null && !configuration.isPlaceHolderForced()) {
                 return result.getResult();
             } else {
-                return computeAsyncPlaceholder(response, configuration.isCount());
+                // We wrap the async placeholder in a listitem which makes it more consistent with the 
+                // result of the macro.
+                return String.format("<li>%s</li>", computeAsyncPlaceholder(response, configuration.isCount()));
```

Comment:
Are you sure this won't lead to nested `<li>` elements once the placeholder has been resolved?